### ISSUE TITLE
[css-contain-2] What does it actually mean for an element to be selected?

### ISSUE THREAD

**Issue by Psychpsyo** [w3c/csswg-drafts#11297]
https://www.w3.org/TR/css-contain-2/#relevant-to-the-user says that something is relevant to the user if it or its contents are selected.
It then links to https://www.w3.org/TR/css-pseudo-4/#highlight-pseudos as an explanation of what selected means.
That link explains how to style things like ::selection or ::target-text and gives a rough description of what they apply to, but it doesn't specifically define selectedness.
Is the intended meaning here that anything that ::selection applies to is relevant to the user?
Or is it any highlight pseudo?
Or is there a different definition of selectedness that used to be here but got moved elsewhere?

TL;DR: What does it mean for an element to be selected for the purposes of relevancy to the user?

**Comment by Psychpsyo** [comment]
Actually, this has been resolved in the latest version.
My mistake for looking at the TR version. -.-
